example/src/main/java/org/aerogear/mobile/example/ui/SecurityServiceFragment.java

```diff
@@ -178,7 +178,10 @@ public void detectBackupEnabled(Map<String, SecurityCheckResult> results) {
      */
     public void detectDeviceEncryptionStatus(Map<String, SecurityCheckResult> results) {
         totalTests++;
-        //TODO: add check
+        SecurityCheckResult result = securityService.check(SecurityCheckType.HAS_ENCRYPTION_ENABLED);
+        if (result.passed()){
+            setDetected(deviceEncrypted, R.string.device_encrypted_positive);
```

Comment:
Is this correct, isn't encryption enabled a good thing? At the moment I'm seeing that encryption is enabled but it's in red writing.

```java
if (!result.passed()) {
    setDetected(deviceEncrypted, R.string.device_encrypted_negative);
}
```

would make more sense to me.